arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -879,6 +1096,62 @@
       },
       "description": "Redis cache access keys."
     },
+    "RedisFirewallRule": {
+      "description": "A firewall rule on a redis cache has a name, and describes a contiguous range of IP addresses permitted to connect",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "resource ID (of the firewall rule)"
+        },
+        "name": {
+          "type": "string",
+          "description": "name of the firewall rule"
+        },
+        "type": {
+          "type": "string",
+          "description": "type (of the firewall rule resource = 'Microsoft.Cache/redis/firewallRule')"
+        },
+        "properties:": {
```

Comment:
are you missing flattening for this properties? https://travis-ci.org/Azure/azure-rest-api-specs/jobs/206299814#L2564